.github/workflows/publish-docs.yml

```diff
@@ -15,12 +15,16 @@ on:
 env:
   RUST_TOOLCHAIN_VERSION: "1.82.0"
```

Comment:
Should this also be bumped?

```suggestion
  RUST_TOOLCHAIN_VERSION: "1.84.1"
```

Reply:
Good spotting! I will update the issue template to include this.